api/grpcserver/globalstate_service.go

```diff
@@ -348,9 +337,6 @@ func (s *GlobalStateService) AppEventStream(
 	*pb.AppEventStreamRequest,
 	pb.GlobalStateService_AppEventStreamServer,
 ) error {
-	// TODO: implement me! We don't currently have any app events
-	// See https://github.com/spacemeshos/go-spacemesh/issues/2074
-
 	return status.Errorf(codes.Unimplemented, "this endpoint has not yet been implemented")
```

Comment:
Maybe let's just remove it from the proto? It's never been implemented so it wouldn't really be breaking anything...

Reply:
Created https://github.com/spacemeshos/go-spacemesh/pull/6708